operator/src/main/java/oracle/kubernetes/operator/KubernetesConstants.java

```diff
@@ -22,6 +22,14 @@ public interface KubernetesConstants {
   String DOMAIN_VERSION = "v9";
   String OLD_DOMAIN_VERSION = "v8";
 
+  String CLUSTERS_CRD_NAME = "clusters.weblogic.oracle";
```

Comment:
What is this used for? I was going to say that I would like the Clusters resource to be in the same group as the Domain resource -- I think that you are doing this based on what I see below.

Reply:
This is the name of the CRD for Cluster kind.  Cluster Resources are still part of the same group ans the Domain resource.

bash-4.2$ kubectl get crds
NAME                       CREATED AT
clusters.weblogic.oracle   2022-05-19T17:29:48Z